ncm-spma/src/main/pan/components/spma/yumng/schema.pan

```diff
@@ -0,0 +1,73 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template components/spma/yumng/schema;
+
+include 'components/spma/schema';
+
+type SOFTWARE_GROUP = {
+    "default" : boolean = true
+    "mandatory" : boolean = true
+    "optional" : boolean = false
+    "names" : string[]
```

Comment:
i've probably missed the discussion why a dict is not ok for yumng, but  /software/groups ideally has the same type for all backends.